src/nodes/functions/material/getGeometryRoughness.js

```diff
@@ -1,9 +1,9 @@
-import { normalGeometry } from '../../accessors/NormalNode.js';
+import { normalView } from '../../accessors/NormalNode.js';
 import { Fn } from '../../shadernode/ShaderNode.js';
 
 const getGeometryRoughness = Fn( () => {
 
-	const dxy = normalGeometry.dFdx().abs().max( normalGeometry.dFdy().abs() );
+	const dxy = normalView.dFdx().abs().max( normalView.dFdy().abs() );
```

Comment:
I've noticed that a lot of code relies on normalView for lighting calculations and PBR purposes. However, we can't depend on it as-is because, as far as I understand, normalView is only assigned once using once() and is never reassigned.

This creates a limitation where the node-based approach doesn't seem to fully support normalNode. For example, in the current PR, roughness calculations would only work if there's a roughnessMap. Otherwise, it defaults to using normalView, which is derived from normalLocal rather than normalNode.

If my understanding is correct, normalNode effectively resolves to normalView, making normalNode incompatible with the current node system. Or am I missing something here, @sunag?